openvpn/cmd_wrapper_test.go

```diff
@@ -32,10 +33,12 @@ func TestHelperProcess(t *testing.T) {
 
 func TestWaitAndStopProcessDoesNotDeadLocks(t *testing.T) {
 	execTestHelper := NewExecCmdTestHelper("TestHelperProcess")
-	execCommand := execTestHelper.ExecCommand
+	execCommand := func(arg ...string) *exec.Cmd {
+		return execTestHelper.ExecCommand("openvpn", arg...)
+	}
 	execTestHelper.AddExecResult("", "", 0, 10000, "openvpn")
 
-	process := NewCmdWrapper("openvpn", "[process-log] ", execCommand)
+	process := NewCmdWrapper("[process-log] ", execCommand)
```

Comment:
we could make "[process-log]" as prefix, or use global one from openvpn package.

Reply:
moved the prefixes to constants.